src/simages/main.py

```diff
@@ -250,10 +251,8 @@ def cli():
         elif args["show"]:
             show(db)
         elif args["find"]:
-            # dups = find(db, match_time=args["--match-time"])
-            dups = find_pairs(args["<path>"], db=db, epochs=int(args["--epochs"]))
-            # Add similar images
-            # sims = find_similar(db)
+            pairs = int(args["--pairs"]) if args["--pairs"] else 10  # Default to 10 pairs if no value is provided
```

Comment:
Looks good, also: we want to set the default for —pairs by adding it to argparser, otherwise will get a key error.